ivy/functional/frontends/tensorflow/layers.py

```diff
@@ -0,0 +1,6 @@
+#global
+import ivy
+import ivy.functional.frontends.tensorflow as tf_frontend
+
+def tf_frontend.keras.layers.concatenate(value, axis, name="ivy.functional.frontends.tensorflow.keras.layers.concatenate"):
+    return tf_frontend.keras.layers.concatenate(value, axis)

```

Comment:
please read the deep dive documentation for details on how to implement frontend functions properly(using only ivy functions etc.): https://lets-unify.ai/ivy/deep_dive.html